HsYAML-aeson.cabal

```diff
@@ -30,12 +30,16 @@ source-repository head
 library
   exposed-modules: Data.YAML.Aeson
   build-depends:
-    , HsYAML      ^>= 0.1.1.0
+    , HsYAML      ^>= 0.2
     , aeson       ^>= 1.4.0.0
     , base         >= 4.5 && < 4.13
     , bytestring  ^>= 0.9.2.1 || ^>= 0.10.0.2
+    , containers   >=0.4.2 && <0.7
+    , ordered-containers
```

Comment:
missing version constraints for `ordered-containers`; but note that HsYAML-aeson currently supports base-4.5; and ordered-containers only supports base-4.5 in version 0.1.* (see https://matrix.hackage.haskell.org/#/package/ordered-containers)

what are we using `ordered-containers` for?

Reply:
forgot to remove it